src/subgraph/common/index.ts

```diff
@@ -1,9 +1,11 @@
 import { Chain } from '@parifi/references';
 
 export const publicSubgraphEndpoints: { [key in Chain]: string } = {
+
   [Chain.ARBITRUM_SEPOLIA]: 'https://subgraph.satsuma-prod.com/ac10c1d41dcb/parifis-team--3804602/parifi-snx-sepolia/api',
```

Comment:
Use .env for private subgraph endpoints. If public subgraph endpoint is not available, set it to empty string